lib/codeowners/cli/main.rb

```diff
@@ -20,6 +20,10 @@ class Main < Base
       option :interactive, default: true, type: :boolean, aliases: '-i'
       option :validateowners, default: true, type: :boolean, aliases: '-v'
       option :autocommit, default: false, type: :boolean, aliases: '-c'
+      option :check_unrecognized_line, default: true, type: :boolean
```

Comment:
I'd remove the `check_` from the prefix as the tool already has checker in the name and the main command is check. So, sounds clear use `codeowners-checker check --unrecognized-line`

Also, generally bash commands are divided by dashes and not underscores, not sure if we can make it with dashes like `--unrecognized-line` instead of `--unrecognized_line` 🤔

Reply:
ah! now I saw that the conversion is automatic from `-` to `_` :)